jerry-debugger/jerry-client.py

```diff
@@ -0,0 +1,22 @@
+import socket
+import sys
+
+port = 5001 
+
+def main():
+	try:
+		client_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+		client_socket.connect(("localhost", port))
+	except socket.error, msg: 
+		print 'Failed to create the socket. Error code: ' + str(msg[0]) + ', Error message: ' + msg[1]
+		sys.exit();
+
+	print 'Socket created on:', port
+
+	data = client_socket.recv(512)
+	print "RECIEVED:" , data
```

Comment:
typo: "received"
moreover, this print won't work with python 3, use print("RECEIVED: %s" % data) instead (if data is a string).